Shared/Tests.Sync.Shared/PermissionsTests.cs

```diff
@@ -47,5 +50,35 @@ public void PermissionChange_ShouldNotBeInDefaultSchema()
         {
             Assert.That(RealmSchema.Default.Find(nameof(PermissionChange)), Is.Null);
         }
+
+        [Test, Explicit]
```

Comment:
It works on iOS simulators. Whenever you want to execute the test, you'll have to update the server url to wherever your server is located. This will be updated once we get the testing framework running, so I haven't overdesigned it.